apps/event-service/app.py

```diff
@@ -25,7 +26,25 @@ def get_events():
     res = supabase.table("events").select(
         "name,description,start,end,id").execute()
     return res.data
-
+  
+  
+@app.get("/events/<day>")
+def get_events_by_day(day: int):
+  day_begin, day_end = None, None
+  one_day = timedelta(1)
+  
+  if day == "1":
+    day_begin = datetime(2023,2,3)
+  elif day == "2":
+    day_begin = datetime(2023,2,4)
+  elif day == "3":
+    day_begin = datetime(2023,2,5)
```

Comment:
what happened to using a for loop instead? and what if day is not 1 2 or 3? please avoid redundant if-elses